sos/report/plugins/__init__.py

```diff
@@ -1570,9 +1570,9 @@ def add_blockdev_cmd(self, cmds, devices='block', timeout=300,
                              tags=_dev_tags)
 
     def add_tape_cmd(self, cmds, devices='tape', timeout=300,
-                         sizelimit=None, chroot=True, runat=None, env=None,
-                         binary=False, prepend_path=None, whitelist=[],
-                         blacklist=[], tags=[]):
+                     sizelimit=None, chroot=True, runat=None, env=None,
+                     binary=False, prepend_path=None, whitelist=[],
+                     blacklist=[], tags=[]):
```

Comment:
This hunk of this patch seems to be fixing up some whitespace errors in the previous patch? It's better to just re-do the patches to avoid this kind of cross-patch dependency (`git rebase -i`).

Reply:
Sorry, I jumped between the RHEL 7 server where I have the tapes, to test and code this, and the usual server where I send the PRs, and I noticed too late the mix